examples/cms-buttercms/package.json

```diff
@@ -1,21 +1,28 @@
 {
+  "name": "nextjs-starter-buttercms",
```

Comment:
```suggestion
```

This might have been it - we have a script that ensures we have similar `package.json` files 👍